rest/client/rest.go

```diff
@@ -555,7 +555,7 @@ func (c *communicatorImpl) GetClientConfig(ctx context.Context) (*evergreen.Clie
 
 func (c *communicatorImpl) GetSubscriptions(ctx context.Context) ([]event.Subscription, error) {
 	info := requestInfo{
-		path:    "/subscriptions",
+		path:    fmt.Sprintf("/subscriptions?owner=%s&type=person", c.apiUser),
```

Comment:
In the event that the user has configured authentication, what happens here?

Reply:
What do you mean exactly? This should use the user specified in the .evergreen.yml file

Reply:
Sorry, that should have read 'has not', ex: new employee, hasn't setup their evergreen.yml file, tries to use this or another command like it. Will it interpolate an empty string, or will it fail earlier than this?

Reply:
If the .evergreen.yml file doesn't exist the CLI can't communicate with the app server at all. If the user isn't in crowd yet, the request will noop before getting to the validation that checks this

Reply:
1. Got this error with my valid config: error fetching subscriptions: server returned error while fetching subscriptions: Invalid owner type. What's going on there?

2. I touched an empty file, and supplied that to this evergreen binary. `touch empty.yml && ./clients/darwin_amd64/evergreen -c empty.yml subscriptions list` which gave me ' unsupported protocol scheme ""'. This error is because the api url is empty. THis has nothing to do with your commit, but more broadly indicates that we need a bit more validation for the cli communicators, so i'm going to ticket this.

Reply:
The first error is fixed by a route change in EVG-3375 (not merged yet)